php/libraries/NDB_BVL_InstrumentStatus_ControlPanel.class.inc

```diff
@@ -156,21 +156,17 @@ class NDB_BVL_InstrumentStatus_ControlPanel extends NDB_BVL_InstrumentStatus
         if (!$hasAccess) {
             return false;
         }
-
         if (isset($_POST['ClearInstrument'])) {
-            if (isset($_POST['deleteconfirm']) && isset($_POST['deleteconfirm2']) ) {
-                $user = User::singleton();
-                if ($user->hasPermission('send_to_dcc')) {
-                    $instrument = \NDB_BVL_Instrument::factory(
-                        $this->testname,
-                        $this->_commentID,
-                        $this->subtest ?? ''
-                    );
-                    $instrument->clearInstrument();
-                    return false;
-                } else {
-                    return false;
-                }
+            header("Refresh: 0;");
```

Comment:
This should not be using functions that create side effects on the output stream like `header`.

Reply:
Where should the refresh statement be?

Reply:
It should not be.

Reply:
I deleted the reset header. This issue is outside of this PR's scope as you mentioned above.